test/driver.js

```diff
@@ -673,3 +673,5 @@ var Driver = (function DriverClosure() {
 
   return Driver;
 })();
+
+exports.Driver = Driver;
```

Comment:
Without any changes to this file, the `no-unused-vars` rule would complain since `Driver` isn't called from within this file (and this seemed better than adding a bunch more `// eslint-disable-line no-unused-vars`).

Please keep in mind here that this file, as with most/all test files, are run using Node.js where `exports` is defined. Note how https://github.com/mozilla/pdf.js/blob/master/test/.eslintrc lists `node` as one of the environments, and test files such as e.g. https://github.com/mozilla/pdf.js/blob/master/test/downloadutils.js#L170-L171 already uses `exports`.

I really ought to have mentioned this since it's not that obvious, but hopefully the above clears it up!